Process.h

```diff
@@ -106,19 +108,7 @@ typedef struct ProcessMergedCommand_ {
 
 typedef struct Process_ {
    /* Super object for emulated OOP */
-   Object super;
-
-   /* Pointer to quasi-global data */
-   const struct Machine_* host;
-
-   /* Process identifier */
-   pid_t pid;
-
-   /* Parent process identifier */
-   pid_t ppid;
-
-   /* Thread group identifier */
-   pid_t tgid;
+   Row super;
```

Comment:
The `Process` class should not inherit the `Row` class. Make it a composition relation instead.
You know in OOP, the inheritance is better used for "is a kind of" relation. A inherits B when A is a kind of B.

Reply:
I think I'm still confused about the purpose of the `Row` class here. Because in the current PR, the `Row` object contains members that describe tree relations - the tree releations may not be applicable to future things that could derive from this `Row` class. Such members shouldn't belong to the `Row`, AFAICT.

Is the `Row` class supposed to be like a `ListBoxItem`, which manages the display logic and the handler routine upon mouse click? Or is it supposed to be something else?

Reply:
As mentioned elsewhere, I'm with @BenBE on this one.  Leaving this conversation as 'open' for now though, until the next classes using Row & Table (beyond Process & ProcessList) arrive.